csharp/extractor/Semmle.Extraction.CSharp/Extractor/Context.cs

```diff
@@ -267,7 +267,7 @@ private void Populate(ISymbol? optionalSymbol, Entities.CachedEntity entity)
 
             bool duplicationGuard, deferred;
 
-            if (ExtractionContext.Mode is ExtractorMode.Standalone)
+            if (ExtractionContext.IsStandalone)
```

Comment:
Note, that this is not semantically equivalent! But the original code looks like a mistake.